Makefile

```diff
@@ -62,9 +62,18 @@ oci-validate-json: validate.go
 oci-validate-examples: cmd/oci-validate-examples/main.go
 	go build ./cmd/oci-validate-examples
```

Comment:
I am thinking of moving the markdown extraction to a markdown_test.go invocable via `go test` and not have a separate binary.